<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="SelTrajectorypushButton">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>8</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Trajectory ...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="TrajectorylineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>13</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelSimbNIBSpushButton">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>34</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select SimbNIBS ...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="SimbNIBSlineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>39</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="T1WlineEdit">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>66</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelT1WpushButton">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>61</y>
     <width>135</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select T1W ...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ContinuepushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>138</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>CONTINUE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelTProfilepushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>135</width>
     <height>56</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Thermal
profile ...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ThermalProfilelineEdit">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>150</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CTcheckBox">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>98</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use CT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CTlineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="cursorPosition">
    <number>3</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SelCTpushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>95</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
